scripts/diff_gas_profiles.py

```diff
@@ -32,7 +32,7 @@ class Profile:
 def _read_profile(filename: str) -> Profile:
     contract = None
     profile = Profile(filename=filename)
-    with open(filename) as fp:
+    with open(filename, encoding="utf-8") as fp:
```

Comment:
Why adding this change and then removing this later in the series?
If this hunk is not meant to be in the PR at all, then it just adds noise and should be removed.